sql/sys_vars.cc

```diff
@@ -382,14 +382,14 @@ static Sys_var_charptr Sys_basedir(
        READ_ONLY GLOBAL_VAR(mysql_home_ptr), CMD_LINE(REQUIRED_ARG, 'b'),
        IN_FS_CHARSET, DEFAULT(0));
 
-static Sys_var_charptr sys_temporal_current_timestamp(
-       "temporal_current_timestamp", "Default AS OF value for versioned tables",
-       GLOBAL_VAR(temporal_current_timestamp), CMD_LINE(REQUIRED_ARG, 'b'),
+static Sys_var_vers_asof Sys_vers_current_time(
```

Comment:
It was [decided](https://github.com/tempesta-tech/mariadb_10.2/issues/117) to use `temporal_current_timestamp` name for variable.